frontend/src/app/search.tsx

```diff
@@ -458,22 +458,20 @@ export default class Search extends Component {
 
   private buildSearchUrl() {
     let backend = "/modules.json?search="
-    if (this.state.input != null && this.state.input !== "") {
-      backend += this.state.input
```

Comment:
This is needed to pass the parameter down to the search. Currently we get:
```
/modules.json?search=&key=&direction=desc&offset=0&limit=10
```
that should include the input after search= for example
search="hello"

Reply:
very good catch :+1:

Reply:
I found another problem as a result of fixing this.